src/System.Collections/tests/Generic/Comparers/Comparer.Generic.Tests.cs

```diff
@@ -32,24 +32,27 @@ public void Comparer_EqualsShouldBeOverriddenAndWorkForDifferentInstances()
             Assert.False(comparer.Equals(3));
             Assert.False(comparer.Equals("foo"));
             Assert.False(comparer.Equals(Comparer<Task<T>>.Default));
+        }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetcoreUwp, "")]
```

Comment:
Add the bug number here:
`[SkipOnTargetFramework(TargetFrameworkMonikers.NetcoreUwp, "See dotnet/corert#1736")]`